frontend/national-boxing-championships-explorer/style.css

```
@@ -0,0 +1,1608 @@
+
+@keyframes slideUp {
+  from { opacity: 0; transform: translateY(40px); }
+  to { opacity: 1; transform: translateY(0); }
+}
+
+@keyframes scaleIn {
+  from { opacity: 0; transform: scale(0.9); }
+  to { opacity: 1; transform: scale(1); }
+}
+
+@keyframes pulseGlow {
```

Stylelint (17.14.0)

[error] Expected keyframe name "pulseGlow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)